### PR TITLE
02 - MVC - Controller

### PR DESCRIPTION
**Pull request by mar7in91Wolox**
# Summary
- I manually added the _HelloWorldController.cs_ into the folder _src/Controllers_ by following this [tutorial](https://docs.microsoft.com/en-us/aspnet/core/tutorials/first-mvc-app-xplat/adding-controller?view=aspnetcore-2.1)
- I could see the return message of Index method.

# Trello Card
https://trello.com/c/Vtj1EJhi

# Screenshots
image

### CODE REVIEW

File: training-net/src/Controllers/HelloWorldController.cs [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+        // 
+        // GET: /HelloWorld/Welcome/ 
+
+        public string Welcome()
+        {
+            return "This is the Welcome action method...";
+        }
+    }
+}
```
Comment (truccomarcos) [inline training-net/src/Controllers/HelloWorldController.cs]: Make sure to add an empty line here :slightly_smiling_face:

File: training-net/src/Controllers/HelloWorldController.cs [hunk 2]
```diff
@@ -0,0 +1,24 @@
+using Microsoft.AspNetCore.Mvc;
+using System.Text.Encodings.Web;
+
+namespace MvcMovie.Controllers
+{
+    public class HelloWorldController : Controller
+    {
+        // 
+        // GET: /HelloWorld/
```
Comment (truccomarcos) [inline training-net/src/Controllers/HelloWorldController.cs]: Do you think this is a necessary comment? Or would it be best if we removed it?

File: training-net/src/Controllers/HelloWorldController.cs (hunk 1 above)
Comment (mar7in91Wolox) [reply training-net/src/Controllers/HelloWorldController.cs]: Done

File: training-net/src/Controllers/HelloWorldController.cs (hunk 2 above)
Comment (mar7in91Wolox) [reply training-net/src/Controllers/HelloWorldController.cs]: Done

**Pull request merged**